frontend/pulse3d/src/components/uploadForm/FileDragDrop.js

```diff
@@ -46,22 +47,30 @@ export default function FileDragDrop({
   dropZoneText = "Drop Here",
   containerStyle = {},
   dropZoneStyle = {},
+  setResetDragDrop,
+  resetDragDrop,
 }) {
+  useEffect(() => {
+    if (resetDragDrop) setResetDragDrop(false);
+  }, [resetDragDrop]);
+
   return (
     <Container style={containerStyle}>
-      <FileUploader
-        hoverTitle=" "
-        handleChange={handleFileChange}
-        name="file"
-        types={fileTypes}
-        multiple={true}
-      >
-        <DropZone style={dropZoneStyle}>
-          {dropZoneText}
-          <br />
-          <FileSelectionLabel>[ {fileSelection || "No file selected"} ]</FileSelectionLabel>
-        </DropZone>
-      </FileUploader>
+      {!resetDragDrop && (
```

Comment:
To reset the state of the component it is enough to just re-render it?

Reply:
The conditional is essentially removing and then re-appending to the DOM again so it's force resetting the internal state of the file-drag-drop-files component